src/Tribe/Main.php

```diff
@@ -351,6 +354,9 @@ public function bind_implementations() {
 		// Attendee Registration Page
 		tribe_register_provider( 'Tribe__Tickets__Attendee_Registration__Service_Provider' );
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: ac77f10040d9aeebc17f3362070073e3</sub>